docs/SetupDatabaseForAWS.md

```diff
@@ -0,0 +1,31 @@
+# Set up a database for Scalar DB/Scalar DL deployment on AWS
+
+This guide explains how to set up a database for Scalar DB/Scalar DL deployment on AWS.
+
+## DynamoDB
+
+Amazon DynamoDB is available for use in AWS by default. You do not need to set up anything manually to use it.
+
+### Optional steps
+
+* Configure advanced monitoring services with the [Azure official guide](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/monitoring-automated-manual.html), monitoring is enabled in DynamoDB by default.
+* You can change the `Read/Write Capacity` of DynamoDB tables based on your requirements.
```

Comment:
Don't say `You can`. It's unclear if it is `you should` or `you don't have to`.
It's a manual guide for production, so you should, right?

Any docs explaining the best practice for this configuration?
We should refer.

Reply:
I have updated the statement and changed `You can` to `You should`.

>Any docs explaining the best practice for this configuration?
We should refer.
 I could not find any document which specified updating Read/Write capacity for DynamoDB.
 It is mentioned  as part of other documents related to auto-scaling and  throughput.